### PR TITLE
CFL minor corrections

### PR DESCRIPTION
**Pull request by mpentek**
No change in functionality

### CODE REVIEW

File: applications/FluidDynamicsApplication/python_scripts/cfl_output_process.py [hunk 1]
```diff
@@ -161,10 +154,7 @@ def _EvaluateCFL(self):
         for elem in self.model_part.Elements:
             local_cfl.append(elem.GetValue(KratosMultiphysics.CFL_NUMBER))
 
-        if (self.model_part.GetCommunicator().TotalProcesses() > 1):
-            local_cfl = self.model_part.GetCommunicator().GetDataCommunicator().GathervDoubles(local_cfl, 0)
-        else:
-            local_cfl = [local_cfl]
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/python_scripts/cfl_output_process.py:167]: I was wondering if this could be avoided. :+1:

**Comment by mpentek** [comment]
As far as naming goes,

> Understood. I'd rather call it cfl_output_lower_bound or something similar. I find that cfl_threshold is a bit misleading.

It also does not bound anything, it is just a threshold or limit value or custom limit (would that be better) for the _Summarize function.

**Comment by mpentek** [comment]
@rubenzorrilla I tried to address both redundant code and naming issue.

**Review by rubenzorrilla — APPROVED** [review]
Thanks @mpentek !

**Pull request merged**